tests/pipelines/marigold/test_marigold_normals.py

```diff
@@ -139,6 +139,38 @@ def get_dummy_inputs(self, device, seed=0):
         }
         return inputs
 
+    def test_attention_slicing_forward_pass(self):
+        self._test_attention_slicing_forward_pass(test_mean_pixel_difference=False)
+
+    def test_marigold_normals(self):
```

Comment:
@toshas, I added a most basic fast test for both pipelines just as an example. Can you add more?
fast tests run on every PR and we will not merge unless they all pass. Slow tests on the other hand, only catch errors afterwards and very often flaky and not reliable

Reply:
Thanks @yiyixuxu , I added more smoke tests